pkg/codegen/internal/test/Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY = update_random update_aws update_azure update_azure_native update_kubernetes
+
+RANDOM_SCHEMA       := "https://raw.githubusercontent.com/pulumi/pulumi-random/master/provider/cmd/pulumi-resource-random/schema.json"
```

Comment:
Is the choice for a floating reference (master) as opposed to a fixed reference (v1.2.3.) intentional here?

Does it conflict with choosing to pin versions of packages in a Go file above?

Reply:
No. This has changed to a fixed version.